docs/rpc.md

```diff
@@ -174,10 +174,12 @@ These methods provide the ability of *historical* calls and accept block hash or
 block index or stateroot hash as the first parameter and the list of parameters
 that is the same as of `invokecontractverify`, `invokefunction` and
 `invokescript` correspondingly. The historical call assumes that the contracts'
-storage state has all its values got from MPT with the specified stateroot and
-the transaction will be invoked using interop context with block of the specified
-height. This allows to perform test invocation using the specified past chain
-state. These methods may be useful for debugging purposes.
+storage state has all its values got from MPT with the specified stateroot (or,
+which is the same, with the stateroot of the block of the specified height) and
+the transaction will be invoked using interop context with block which is next to
+the block with the specified height. This allows to perform test invocation using
```

Comment:
"using an interop context of the next block"?

Reply:
Not exactly, context's storage is taken from the current block.